src/Model.h

```diff
@@ -50,7 +51,9 @@ typedef struct PERSON {
 	unsigned short int dct_start_time, dct_end_time, dct_trigger_time, dct_test_time; //digital contact tracing start and end time: ggilani 10/03/20
 	int ncontacts; //added this in to record total number of contacts each index case records: ggilani 13/04/20
 
-} person;
+	bool isAlive() const;
+	bool isDead() const;
```

Comment:
Can we use snake_case for method names please?  (so is_alive() and is_dead()).

Reply:
No problem. Done. :smile: